libindy/src/services/metrics/mod.rs

```diff
@@ -40,25 +42,48 @@ impl MetricsService {
         self.executed_commands_count.borrow_mut()[command_index as usize] += 1;
         self.executed_commands_duration_ms.borrow_mut()[command_index as usize] += duration;
     }
-    pub fn cmd_name(index: usize) -> String {
-        CommandIndex::from(index).to_string().to_case(Case::Snake)
+
+    pub fn cmd_name(index: usize) -> (String, String) {
+        let cmd = CommandIndex::from(index).to_string().find("Command");
+        match cmd {
+            None => (CommandIndex::from(index).to_string().to_owned(), String::from("")),
+            Some(cmd) => (CommandIndex::from(index).to_string()[0..Some(cmd).unwrap()].to_owned(), CommandIndex::from(index).to_string()[Some(cmd).unwrap()+7..].to_owned()) ,
```

Comment:
I don't think string search and manipulations (with magic `+7` offsets and unwraps all over the place) are good idea here. Why not just modify `CommandIndex` to have functions `command_name()` and `command_subname()`?